### ISSUE TITLE
How to load n number of random documents of collection?

### ISSUE THREAD

**Issue by asasi007** [prescottprue/react-redux-firebase#800]
useFirestoreConnect([
    {
      collection: 'words',
      where: ['createdBy', '==', auth.uid],
      limit:10,
    }
  ])

**Comment by prescottprue** [comment]
Not sure what you are asking here - they query you provided should work to load 10 documents from the words collection. Are you looking for that to be a random 10 or are you saying this query doesn't work?

**Comment by AnurajRicky** [comment]
Yes this is working fine! and my question is how to get the random ten documents of from collection?

**Comment by asasi007** [comment]
Yes, this is working fine! my question is how to load ten random documents from a collection?

**Comment by prescottprue** [comment]
Since that is not something directly supported by the Firebase API as a method, it isn't a feature here either.

For something like this it may be more advantageous to go with a Cloud Function that generates a random list of words by querying all the words and selecting a random set (maybe using [javascripts Math.random](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random)). That could then be returned to the client. This can be especially helpful if you are hoping to only return certain data to the client from the documents since you can pluck that data off in your cloud function as well.
